bon-jova-app/pom.xml

```diff
@@ -18,7 +18,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-        <quarkus.platform.version>3.6.4</quarkus.platform.version>
+        <quarkus.platform.version>3.9.4</quarkus.platform.version>
```

Comment:
This can be 3.9.5

Reply:
Actually, 3.10.0 is out

Reply:
Yeah, I tried 3.10.0, and the dev ui didn't work, so I rolled back and did a binary chop to find the point where the dev ui stopped working, which was 3.9.1.